src/pages/FollowUpAnalysis/FollowUpAnalysis.js

```diff
@@ -61,12 +62,15 @@ const FollowUpAnalysis = ({ sectionKey }) => {
             i18n.t('An unexpected error happened during analysis'),
         { critical: true }
     )
-    const [
-        fetchFollowUpList,
-        { loading: loadingFollowUpList },
-    ] = useDataMutation(followUpAnalysisMutation, {
-        onComplete: response => {
-            const elements = response.map(convertElementFromApiResponse)
+    const {
+        loading: loadingFollowUpList,
+        refetch: fetchFollowUpList,
+    } = useDataQuery(query, {
+        lazy: true,
+        onComplete: data => {
+            const elements = data.followups.followupValues.map(
+                convertElementFromApiResponse
+            )
             setElements(elements)
```

Comment:
We should get rid of this pattern. But I suppose that's not part of this PR. Would be good to make sure there's an issue for this though.

Reply:
:100: - I'll create a jira issue and make a PR to remove it this pattern from the various components that use it once I've written some cypress tests

Reply:
Jira issue: https://jira.dhis2.org/browse/DHIS2-11521

Reply:
Nice, thanks!